### DISCUSSION TITLE
More compact version of results screen for score sharing

### DISCUSSION THREAD

**Discussion by epln** [ppy/osu#30045] [Ideas] [answered]
## Current Problem

The score results screen is too large (scrolling is required to view all stats without UI scaling). This makes it slightly annoying to take a screenshot of a performance you want to post on r/osugame or just share a performance to someone without having to send the entire replay, because you have to either:
- Adjust UI scaling (which requires you to go into the settings, turn it down, take the screenshot, go into the settings again, and turn it back up again). This makes everything extremely small and hard to read. Or
- Take 2 screenshots and send them both, with most stats then being visible on both of them.

You could also of course just take a single one, but then one of two things wouldn't be visible because of how the UI is designed:
- Statistics (including the UR and avg. hit error), or
- Performance Breakdown (including the achieved and maximum PP, and the aim, speed, acc, and potentially FL weights.)

To me personally both of these are really important (UR coming from stable, and the weight breakdown just being interesting.)

This problem occurred to me after viewing [this](https://www.reddit.com/r/osugame/comments/1f2vm5g/) post on r/osugame, showcasing a score post of a high PP score set on Lazer. The PP weights are visible, however the UR is nowhere to be seen in the post or from osu-bot's pinned comment (since osu-bot assumes UR is included in the screenshot like it is in posts coming from stable)

## Idea

There could be a button on the results screen after playing a map or watching a replay that shows all stats ("Performance Breakdown", "Timing Distribution", "Accuracy Heatmap", and "Statistics") on a singular screen, or while we're already at it just a button that let's you save / copy to clipboard a singular, more compact breakdown of it.

## Alternative Implementation

If such an implementation is too complicated (new layout, button, etc), how about a button that just temporarily adjusts UI scaling to take a screenshot without having to go to the settings? This would make things already much easier and less tedious while still staying not too complicated to implement. This wouldn't solve the small text issue that comes with it though.
___
I know this is a very neat suggestion, but it would make it way easier for people to post Lazer scores on e.g. r/osugame. It may not be an issue now, however it will be later, since more people are gonna play Lazer, set scores, and will then wanna post those.

**Comment by peppy** [comment, marked as answer]
A full redesign is coming with sharing in mind.
